MessageBroker.js

```diff
@@ -1013,6 +1013,14 @@ class MessageBroker {
 					 }
 				}
 			}
+
+			if (msg.eventType === "custom/myVTT/peerReady") {
```

Comment:
We should consider moving to a switch, or returning within the if statements to break out from the logic to reduce execution size

Reply:
I agree, but that's a much larger change to this file than I'm willing to do in this PR. That feels like a separate "cleanup" PR

Reply:
Ahh, I just pulled up the entire file and there is TONS of these. Yea perhaps.